.github/workflows/ci.yml

```diff
@@ -236,8 +236,8 @@ jobs:
         run: |
           # shellcheck disable=SC2129
           echo "CARGO_INCREMENTAL=0" >> "$GITHUB_ENV"
-          echo "RUSTFLAGS=-Zprofile -Ccodegen-units=1 -Cinline-threshold=0 -Clink-dead-code -Coverflow-checks=off -Cpanic=abort -Zpanic_abort_tests" >> "$GITHUB_ENV"
-          echo "RUSTDOCFLAGS=-Zprofile -Ccodegen-units=1 -Cinline-threshold=0 -Clink-dead-code -Coverflow-checks=off -Cpanic=abort -Zpanic_abort_tests" >> "$GITHUB_ENV"
+          echo "RUSTFLAGS=-Zprofile -Cinline-threshold=0 -Clink-dead-code -Coverflow-checks=off -Cpanic=abort -Zpanic_abort_tests" >> "$GITHUB_ENV"
+          echo "RUSTDOCFLAGS=-Zprofile -Cinline-threshold=0 -Clink-dead-code -Coverflow-checks=off -Cpanic=abort -Zpanic_abort_tests" >> "$GITHUB_ENV"
```

Comment:
@nathanwhit was this an attempt to fix the unit test job ? 

I originally though you were adjusting the flags b/c of a new compiler version but that's probably not the case.

I tried by removing all flags and still got the same result. It looks like an out-of-memory issue to me. 

However OOM is also strange b/c the integration-test job also builds creditcoin-node but doesn't OOM.

Reply:
Yeah that was just me trying to see if I could get memory usage down, I've reverted it now.